cogito/putter.go

```diff
@@ -105,10 +106,10 @@ func (putter *ProdPutter) ProcessInputDir() error {
 		}
 	}
 
+	// If the size is 0 after removing the directory containing the chat message, this will be a chat only put.
 	if inputDirs.Size() == 0 {
-		return fmt.Errorf(
-			"put:inputs: missing directory for GitHub repo: have: %v, GitHub: %s/%s",
-			collected, source.Owner, source.Repo)
+		putter.log.Debug("No GitHub repositories in inputs, Cogito will only send to chat")
```

Comment:
If there are 0 directories, Cogito will send only to chat. Any configuration issue will be intercepted earlier (tuned in next PR).